archetype/config/env/production.js

```diff
@@ -1,37 +1,3 @@
-'use strict'
-
-const winston = require('winston')
-
 module.exports = {
 
-  trailpack: {
```

Comment:
Makes sense. But at same time it could be added into docs and removed from default `production` configuration.